acme_diags/driver/default_diags/lat_lon_model_vs_model.cfg

```diff
@@ -392,32 +321,7 @@ plevs = [200.0]
 test_colormap = "PiYG_r"
 reference_colormap = "PiYG_r"
 contour_levels = [-35,-30,-25,-20,-15,-10,-5,5,10,15,20,25,30,35]
-diff_levels = [-20,-15,-10,-8,-6,-4,-2,2,4,6,8,10,15,20]
-
-
-[#]
-sets = ["lat_lon"]
-case_id = "model_vs_model"
-variables = ["U"]
-seasons = ["ANN"]
-plevs = [200.0]
-test_colormap = "PiYG_r"
-reference_colormap = "PiYG_r"
-contour_levels = [-50, -40, -30, -20, -10, -5, 5, 10, 20, 30, 40, 50]
-diff_levels = [-15, -10, -8, -6, -4, -2, -1, 1, 2, 4, 6, 8, 10, 15]
-
-
-[#]
-sets = ["lat_lon"]
-case_id = "model_vs_model"
-variables = ["U"]
-seasons = ["DJF", "MAM", "JJA", "SON"]
-plevs = [200.0]
-test_colormap = "PiYG_r"
-reference_colormap = "PiYG_r"
-contour_levels = [-75, -50, -30, -20, -10, -5, 5, 10, 20, 30, 50, 75]
-diff_levels = [-15, -10, -8, -6, -4, -2, 2, 4, 6, 8, 10, 15]
-
+diff_levels = [-80,-60,-40,-32,-24,-16,-8,-4, 4, 8,16,24,32,40,60,80]
```

Comment:
I didn't actually run a case, but this `diff_levels` don't look right while the `contour_levels`  range  is smaller...

Reply:
@chengzhuzhang : thanks for catching this. I have now corrected this in 75c6fbbabe50b07d2514fc118f151f90d957d182